frontend/src/app/Main.tsx

```diff
@@ -44,7 +44,37 @@ const Main: React.FC<ViewProps> = () => {
       }}
       renderDrawerContent={() => {
         return (
-          <Text style={SpacingStyle.pad(drawerInsets, 16)}>Drawer content</Text>
+          <View style={SpacingStyle.pad(drawerInsets, 16)}>
+            <TouchableOpacity onPress={()=>{}}>
+                <View style={styles.drawerItem}>
+                  <MaterialIcons name="accessible" size={24} color="black" /> 
+                  <Text style={styles.drawerItemText}>ADA Ride Request</Text>
+                </View>
+            </TouchableOpacity>
+            
+            <TouchableOpacity onPress={()=>{}}>
```

Comment:
Use underlayColor={Color.generic.selection} for consistency.

Reply:
This doesn't exist on TouchableOpacity, it exists on TouchableHighlight.

Reply:
Oh, right. Can you use that for consistency?

Reply:
It might look a bit odd. I used TouchableHighlight to begin with. I think TouchableOpacity is better personally

Reply:
Weird, I'm using it elsewhere and it looks fine.